### DISCUSSION TITLE
What's up with www.pouchdb.com and... Scotland independence?

### DISCUSSION THREAD

**Discussion by janzheng** [pouchdb/pouchdb#8650] [General]
I accidentally went to https://www.pouchdb.com/about and anything under the `www` subdomain is about Scotland. What's going on there?!

**Comment by garethbowen** [comment]
For me it's flagged with a security risk. Definitely needs to be fixed.

**Comment by ChetanGoti** [comment]
It looks like hosting configurations issue. Looks like most(any) subdomain for `pouchdb.com` resolves to IP address `173.255.227.207` which looks like Linode server (https://li237-207.members.linode.com/)

It seems to host https://better-together.arandomurl.com as well and which looks like the default virtual host.

So anything that resolves to above IP address, serves the default Better Together site.

----

```
❯ host pouchdb.com
pouchdb.com has address 173.255.227.207

❯ host www.pouchdb.com
www.pouchdb.com is an alias for pouchdb.com.
pouchdb.com has address 173.255.227.207

❯ host www1.pouchdb.com
www1.pouchdb.com is an alias for pouchdb.com.
pouchdb.com has address 173.255.227.207

❯ host www1asdf.pouchdb.com
www1asdf.pouchdb.com is an alias for pouchdb.com.
pouchdb.com has address 173.255.227.207
```
